validphys2/src/validphys/photon/compute.py

```diff
@@ -71,6 +70,9 @@ def __init__(self, theoryid, lux_params, replicas):
         path_to_F2 = theoryid.path / "fastkernel/fiatlux_dis_F2.pineappl.lz4"
         path_to_FL = theoryid.path / "fastkernel/fiatlux_dis_FL.pineappl.lz4"
         self.path_to_eko_photon = theoryid.path / "eko_photon.tar"
+        with EKO.read(self.path_to_eko_photon) as eko:
+            self.q_in = np.sqrt(eko.mu20)
```

Comment:
Why is it called `mu20`? Is it because it's `(mu_0)^2`?

Reply:
Yes